manifests/state-hostdevice-network/0010-hostdevice-net-cr.yml

```diff
@@ -10,5 +10,5 @@ spec:
   "cniVersion":"0.3.1",
   "name":"{{.HostDeviceNetworkName}}",
   "type":"host-device",
-  "ipam": {{.CrSpec.IPAM}}
+  {{.Ipam}}
```

Comment:
We can use `"ipam":{{or .CrSpec.IPAM "{}"}}` here and avoid extra logic in the state. WDYT?

Reply:
Done